### PR TITLE
Automate get-latest.sh

### PR DESCRIPTION
**Pull request by TechPowerAwaits**
The script no longer prompts for user input and it also installs the
built RPM packages.

### CODE REVIEW

**Comment by TechPowerAwaits** [comment]
Need to add GitHub Action to run this script and create releases from it.

**Comment by TechPowerAwaits** [comment]
Apparently, I didn't test my previous changes and most of the files under SOURCES still has the uostomcat name.

**Comment by TechPowerAwaits** [comment]
The GitHub Actions thing can wait.

**Comment by TechPowerAwaits** [comment]
The script appears to require administrator privileges to install the RPM package.

File: get-latest.sh [hunk 1]
```diff
@@ -66,37 +66,44 @@ fi
...
-	# If the version in the spec file has changed we should reset the
-	# release back to 1
-	if [ "$spec_tomcat_version" != "$tomcat_version" ]; then
-		echo "Previous spec file tomcat_version was ${spec_tomcat_version}. Resetting tomcat_patch_version to 1."
-		if ! sed -ri "s/^(\s*%define\s+tomcat_patch_version\s+)[^\s]+\s*/\11/" "${tc_spec_path}"; then
-			echo -e "${color_failure}RPM spec file update failed${color_reset}"
-			exit 1
-		fi
-	fi
+echo -ne "${color_bold}Building updated RPMs [y/n]? ${color_reset}"
```
Comment (108Mbps) [inline get-latest.sh]: This line still seems to display a prompt.

Comment (TechPowerAwaits) [reply get-latest.sh]: Fixed.

**Review by 108Mbps — APPROVED** [review]
👍

**Pull request merged**